### PR TITLE
Fix script file transformer and openssl command issues 

### PR DESCRIPTION
**Pull request by LiliDeng**

### CODE REVIEW

File: microsoft/testsuites/nested/common.py [hunk 1]
```diff
@@ -38,7 +38,19 @@ def _create_cloud_init_iso(
     password: str,
     host_name: str = "l2vm",
 ) -> str:
-    cmd_result = host.execute(f"openssl passwd -6 {password}", sudo=True, shell=True)
+    cmd_result = host.execute(
+        f"openssl passwd -6 {password}",
+        sudo=True,
+        shell=True,
+    )
+    if cmd_result.exit_code != 0:
```
Comment (squirrelsc) [inline microsoft/testsuites/nested/common.py:62]: Add comments above to explain the expected errors.

Comment (squirrelsc) [reply microsoft/testsuites/nested/common.py:62]: Please give an example error message. So if there are other non-zero exit code, others know the `if` branch is not designed for the new situation.

Comment (LiliDeng) [reply microsoft/testsuites/nested/common.py:62]: updated

File: lisa/tools/qemu.py [hunk 2]
```diff
@@ -76,7 +80,7 @@ def create_vm(
             # e.g. QEMU emulator version 1.5.3 (qemu-kvm-1.5.3-175.el7_9.6)
             # on centos 7.9
             try_pcid_flag = self.node.execute(
-                f"{self._qemu_command} -cpu host,pcid=no", sudo=True
+                f"timeout 20 {self._qemu_command} -cpu host,pcid=no", sudo=True
```
Comment (squirrelsc) [inline lisa/tools/qemu.py:85]: Why add time out here? Add comments above.

Comment (LiliDeng) [reply lisa/tools/qemu.py:85]: added

**Review by squirrelsc — APPROVED** [review]
Please don't add more changes into this PR...

**Comment by LiliDeng** [comment]
> Please don't add more changes into this PR...

sure, I have tested the last commit, and it is related change, so I appended it here.

**Pull request merged**